.github/workflows/test.yml

```diff
@@ -197,9 +197,6 @@ jobs:
       OVN_GATEWAY_MODE: "${{ matrix.gateway-mode }}"
     steps:
 
-    - name: Free up disk space
```

Comment:
why is this being removed?

Reply:
because I ran it several times without it and didn't see any failure related, so it saves ~ 2 mins per job. Maybe this is no longer needed :shrug:

Reply:
I would lean on the side of caution here and not remove it. We were almost out of disk space while running tests, and this frees up 10GB:
https://github.com/ovn-org/ovn-kubernetes/pull/1198

Unless you want to run your job and print out the disk space so we can see how much we have left to confirm this is no longer needed, I would just leave it for now.

Reply:
sure, let me remove it